interface/forms/fee_sheet/new.php

```diff
@@ -29,10 +29,11 @@
 require_once("codes.php");
 
 //acl check
-$can_view = acl_check('admin', 'superbill', false, 'view');
-$can_write = acl_check('admin', 'superbill', false, 'write');
-if (!$can_view && !$can_write)
+if (!acl_check('encounters','coding'))
```

Comment:
Will instead use the acl_check_form() function here. This is something we could then add to all the forms to further secure them.